src/IntegrationTests/java/uk/gov/hmcts/reform/sscs/callback/AdjournCaseIt.java

```diff
@@ -61,6 +61,7 @@ public class AdjournCaseIt extends AbstractEventIt {
     public static final String CCD_ABOUT_TO_SUBMIT = "/ccdAboutToSubmit";
     public static final String CCD_MID_EVENT_PREVIEW_ADJOURN_CASE = "/ccdMidEventPreviewAdjournCase";
     public static final String CCD_MID_EVENT_ADJOURN_CASE_POPULATE_VENUE_DROPDOWN = "/ccdMidEventAdjournCasePopulateVenueDropdown";
+    public static final String CCD_MID_EVENT_ADJOURN_CASE_DUE_DATE = "/ccdMidEventAdjournCaseDirectionDueDate";
     public static final String CCD_MID_EVENT = "/ccdMidEvent";
```

Comment:
you can remove this constant if it's no longer used

Reply:
It's still being used by givenCallToMidEventCallbackWithPathYesNoYesThenValidatesDataWithDueDateInPast